shared/src/components/exercise/intro.cjsx

```diff
@@ -35,7 +35,7 @@ ExerciseIntro =  React.createClass
         <span>{TITLES[stepGroup]}</span>
       </h1>
       {getIntroText[stepGroup](project)}
-      <BS.Button onClick={onContinue}>Continue</BS.Button>
+      <a className="btn continue" onClick={onContinue}>Continue</a>
```

Comment:
should we use https://react-bootstrap.github.io/components.html#buttons-tags instead here? i guess maybe it doesnt matter except more consistent and also the role="button" attribute on the resulting html